packages/browser-integration-tests/utils/generatePlugin.ts

```diff
@@ -158,8 +158,14 @@ class SentryScenarioGenerationPlugin {
         // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
         parser.hooks.import.tap(
           this._name,
-          (statement: { specifiers: [{ imported: { name: string } }] }, source: string) => {
-            if (source === '@sentry/integrations') {
+          (statement: { specifiers: [{ imported?: { name: string }; name?: string }] }, source: string) => {
+            // We only want to handle the integrations import if it doesn't come from the @sentry/browser re-export
+            // In that case, we just want to leave it alone
+            if (
```

Comment:
Without this change, this fails for all tests as the build output of webpack includes the import from `@sentry/integrations` by default and tries to "fix" it in there, but it also does not contain an `imported` field so it just fails.